data/remnant missions.txt

```diff
@@ -1716,3 +1725,164 @@ mission "Remnant: Broken Jump Drive job"
+			scene "scene/penguinscene"
+			`	Taely's droning chant of technical details finally seeps through your reverie: "... So from a captain's point of view flying the penguin should be very similar to a normal ship, except she flies herself and doesn't need a crew to run her systems." She pauses, and looks at you. "Your curiosity provided the final ingredients to bring her to life. You should be the first to have one. You'll still have to go through the shipyard to register her and pay the resource cost to balance the logistics, but this first one is reserved for you." She pats the nose of the Penguin and it makes a wooshing noise. After a moment she starts going over the ship with a scanner, leaving you to contemplate in silence.`
+
+event "remnant: penguin"
+	shipyard "Remnant Gascraft"
+		"Penguin"
+
+
+
+mission "Remnant: Face to Maw 2B"
+	name "Take the tentacle to Chilia's engineers on Caelian"
+	description "The tentacle has properties that may be put to use by the engineers on <planet>."
+	to offer
+		has "Remnant: Face to Maw 1B: done"
+	source "Aventine"
+	destination "Caelian"
+	cargo "tentacle" 1
+	on offer
+		conversation
+			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams centuries ago discovered that these tentacles could be used as a devastating weapon in close quarters combat."`
+			`	He pulls out an image of a strange looking rifle and continues to chant, "The personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go visit it on Aventine, they could turn this tentacle into a potent weapon for you."`
+			choice
+				`	"That sounds good."`
+					accept
+				`	"I would rather not."`
+			`	The technician looks disappointed. "Oh well, enjoy your trophy."`
+			decline
+	on complete
+		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
+		payment 250000
+		"remnant chilia" ++
+		outfit "Void Rifle"
+		conversation
+			branch chilia
+				"remnant chilia" > 1
+			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`
```

Comment:
```suggestion
			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who asks you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`
```
`You [...] are met by security, who asks you to wait`

this one's debatable, if you want to be on the safe side, rewrite it to be clearly plural or singular

Reply:
cc @Zireael07